### PR TITLE
Create test issue

### PR DESCRIPTION
**Pull request by futuretea**
Signed-off-by: futuretea <Hang.Yu@suse.com>

### CODE REVIEW

**Comment by futuretea** [comment]
example: https://github.com/futuretea/bot-test/issues/1

File: github-bot/harvester_github_bot/__init__.py [hunk 1]
```diff
@@ -70,10 +73,37 @@ def issue_transfer(form):
...
+        found = False
+        for comment in comments:
+            if comment.body.startswith('Automation e2e test issue:'):
+                app.logger.debug('Automation e2e test issue already exists, not creating a new one')
+                found = True
+                break
+        if not found:
+            app.logger.debug('Automation e2e test issue does not exist, creating a new one')
+            issue_link = '{}/{}#{}'.format(GITHUB_OWNER, GITHUB_REPOSITORY, issue_number)
+            issue_test_title = '[TEST] ' + issue_link
```
Comment (guangbochen) [inline github-bot/harvester_github_bot/__init__.py]: prefer to add the title with `[e2e] copy the original issue title`

File: github-bot/harvester_github_bot/__init__.py [hunk 2]
```diff
@@ -70,10 +73,37 @@ def issue_transfer(form):
...
+            repo_test = g.get_repo('{}/{}'.format(GITHUB_OWNER, GITHUB_REPOSITORY_TEST))
+            issue_test_template_content = repo_test.get_contents(".github/ISSUE_TEMPLATE/test.md").decoded_content.decode()
```
Comment (guangbochen) [inline github-bot/harvester_github_bot/__init__.py:102]: can we append the original issue link to the end of the content with `related issue: https://github.com/harvester/harvester/issues/123`

**Review by guangbochen — APPROVED** [review]
LGTM

**Pull request merged**